### PR TITLE
Cp1214 memory leaks

### LINKED ISSUE #6636: Remove therubyracer?
I think therubyracer is no longer used and can be removed.

### PR DESCRIPTION
**Pull request by juanignaciosl**
Triggering tests. This is related to the memory leaks reported at CartoDB/cartodb-platform#1214. Closes #6636.

### CODE REVIEW

**Comment by juanignaciosl** [comment]
@gfiorav , CR this, please.

@santisaez I've begun doing some memory usage analysis in my development environment to get to know the tooling and take a first look. I haven't found any major leakage around our code yet, but some of the gems were really outdated and had known memory leaks. I've upgraded to the fixing version, quite minor changes. The biggest change is probably removing the dependency to backports, now that we have Ruby 2.2.3.

I want to first deploy this and check the memory evolution.

File: app/models/carto/permission.rb [hunk 1]
```diff
@@ -1,5 +1,4 @@
 require 'active_record'
-require_relative './user'
```
Comment (gfiorav) [inline app/models/carto/permission.rb]: what does this have to do with anything? Are you taking the opportunity to solve something else or is it a typo? :smile:

Comment (juanignaciosl) [inline app/models/carto/permission.rb]: Doing some memory profiling said that this line was causing memory leaks :-\ Since both are ActiveRecord models that line is certainly not needed, maybe the loader was "overloading". I don't actually now yet why and I certainly think it's hard to believe, but this works without it, it might worth a try...

Comment (gfiorav) [inline app/models/carto/permission.rb]: Fascinating!

Comment (juanignaciosl) [inline app/models/carto/permission.rb]: :laughing:

**Comment by gfiorav** [comment]
Ok, looks :+1: but there are some things I don't understand. Let's catch up some time and talk about this!
